### PR TITLE
LBSD-2116 fix: decreased versions for nunjucks packages

### PR DESCRIPTION
**Pull request by eos87**
Since more recent versions of nunjucks use beta versions of babel packages, it breaks the making of new theme
extensions given that we're using stable v6 of babel. Reducing the version of nunjucks packages solves the problem

### CODE REVIEW

File: server/liveblog/themes/themes_assets/default/gulpfile.js [hunk 1]
```diff
@@ -2,7 +2,6 @@
 
 const gulp = require('gulp');
 const browserify = require('browserify');
-const nunjucksify = require('nunjucksify');
```
Comment (tomaszrondio) [inline server/liveblog/themes/themes_assets/default/gulpfile.js]: why is this removed?

Comment (eos87) [reply server/liveblog/themes/themes_assets/default/gulpfile.js]: It's not required anymore
